<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="calendar_widget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>132</y>
     <width>411</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    color: black;  
    background-color: light grey;  
    selection-background-color: rgb(200,200,200); 
    selection-color black; 
    alternate-background-color: rgb(126,126,126); 
font: 12pt;
</string>
   </property>
  </widget>
  <widget class="QListWidget" name="event_list_widget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:16pt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(173,255,147)</string>
   </property>
   <property name="text">
    <string>Crea Evento</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>0</y>
     <width>861</width>
     <height>131</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240,240,240);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Calendario Eventi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>140</y>
     <width>861</width>
     <height>381</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240,240,240);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_name">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>320</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_location">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>350</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_time">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>380</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_description">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>440</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_organizer">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>410</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome Evento</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_location">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>350</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Luogo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>380</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Orario</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_description">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>440</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrizione</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_organizer">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>410</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Organizzatore</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_select">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleziona Evento da Modificare</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>470</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200,200,200)</string>
   </property>
   <property name="text">
    <string>INDIETRO</string>
   </property>
  </widget>
  <zorder>label_title</zorder>
  <zorder>label_background</zorder>
  <zorder>calendar_widget</zorder>
  <zorder>event_list_widget</zorder>
  <zorder>add_button</zorder>
  <zorder>line_edit_name</zorder>
  <zorder>line_edit_location</zorder>
  <zorder>line_edit_time</zorder>
  <zorder>line_edit_description</zorder>
  <zorder>line_edit_organizer</zorder>
  <zorder>label_name</zorder>
  <zorder>label_location</zorder>
  <zorder>label_time</zorder>
  <zorder>label_description</zorder>
  <zorder>label_organizer</zorder>
  <zorder>label_select</zorder>
  <zorder>back_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
